### ISSUE TITLE
Empty Junit output

### ISSUE THREAD

**Issue by manuel-belvo** [semgrep/semgrep#5924]
**Describe the bug**
JUnit XML comes empty, even tough the console did show that tests were executed.

**To Reproduce**
0. Have an app.
1. Run:
```
docker run --rm -v $$(pwd)/src:/opt/src returntocorp/semgrep semgrep --exclude=staticfiles/ --severity=WARNING --severity=ERROR --config=p/security-audit --error --junit-xml /opt/src/app > test-results/semgrep_results.xml
```

**Expected behavior**
Have JUnit output with information, useful for CircleCI.

**Outputs**
JUnit output:
```
<?xml version="1.0" ?>
<testsuites disabled="0" errors="0" failures="0" tests="0" time="0.0">
	<testsuite disabled="0" errors="0" failures="0" name="semgrep results" skipped="0" tests="0" time="0"/>
</testsuites>
```

Console output:
```
...
Fetching rules from https://semgrep.dev/registry.

Scanning across multiple languages:
         python | 91 rules × 469 files
    <multilang> | 10 rules ×  12 files
           json |  1 rule  ×   2 files


Some files were skipped or only partially analyzed.
  Scan skipped: 174 files matching --exclude patterns, 2 files larger than 1.0 MB
  For a full list of skipped files, run semgrep with the --verbose flag.

(need more rules? `semgrep login` for additional free Semgrep Registry rules)

Ran 246 rules on 477 files: 0 findings.
```

**What is the priority of the bug to you?**

- [X] P0: blocking your adoption of Semgrep or workflow

**Environment**
Docker

**Use case**
Enable to keep track of runs on CircleCI

**Comment by r2c-demo** [comment]
This issue is synced in Linear at https://linear.app/r2c/issue/PA-1770/empty-junit-output. _Note_: this link is for r2c use only and is not accessible publicly.

**Comment by brendongo** [comment]
@manuel-belvo i'm not too familiar with junit xml. What output in the junit xml would you expect here? it looks like in the console output there were no findings either and I think our junit xml output lists any found issues.

**Comment by manuel-belvo** [comment]
@brendongo I fond this documentation: https://www.ibm.com/docs/en/developer-for-zos/9.1.1?topic=formats-junit-xml-format 

Here's a schema: https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd

And a nice blog: https://docs.getxray.app/display/XRAY/Taking+advantage+of+JUnit+XML+reports

**Comment by brendongo** [comment]
Junit output is low priority for us but we will gladly help/accept PRs for this!

**Comment by manuel-belvo** [comment]
Is there a current way to integrate semgrep with CircleCI?

**Comment by aryx** [comment]
Yes you can integrate semgrep with circleCI. See https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/
Here is another example we use internally for the semgrep repo itself:
https://github.com/returntocorp/semgrep/blob/develop/.circleci/config.yml
